os.cpp

```diff
@@ -1,30 +1,32 @@
 /*
- * os.cpp
- *
- * Baker, Ballard, Jager-Kujawa
- * CSC 341
- * Spring 2016
- */
+* os.cpp
+*
+* Baker, Ballard, Jager-Kujawa
+* CSC 341
+* Spring 2016
+*/
 
 #include "os.hpp"
 
-queue<User> readyQueue = queue<User>();
-queue<User> blockedQueue = queue<User>();
-User currentUser;
+queue<Process> readyQueue = queue<Process>();
+queue<Process> blockedQueue = queue<Process>();
+Process currentProcess;
 bool semaphore = unlocked;
 
 int sysclock;
 int switchTime;
+userID currentUser;
+Process processTable[64];
```

Comment:
Use NUM_FRAMES instead of constant 64; just in case the number of frames changes later for whatever reason.

NUM_FRAMES is defined in utils.hpp